xline/src/storage/kvstore.rs

```diff
@@ -192,55 +208,112 @@ impl KvStoreBackend {
             .collect()
     }
 
+    /// Sort kvs by sort target and order
+    fn sort_kvs(kvs: &mut [KeyValue], sort_order: SortOrder, sort_target: SortTarget) {
+        match (sort_target, sort_order) {
+            (SortTarget::Key, SortOrder::None) => {}
+            (SortTarget::Key, SortOrder::Ascend) => {
```

Comment:
We can cover these codes from line 215 to line 217 in `test_keys_only` by constructing requests with different `SortOrder`. It can help improve our test coverage.

Reply:
added a  new test